tokio-macros/src/entry.rs

```diff
@@ -443,6 +443,26 @@ pub(crate) fn main(args: TokenStream, item: TokenStream, rt_multi_thread: bool)
     }
 }
 
+// Check whether given attribute is `#[test]` or `#[::core::prelude::v1::test]`.
+fn is_test_attribute(attr: &Attribute) -> bool {
+    let path = match &attr.meta {
+        syn::Meta::Path(path) => path,
+        _ => return false,
+    };
+    let segments = ["core", "prelude", "v1", "test"];
```

Comment:
It's also available as a few other paths such as `::std::prelude::v1::test` and `::core::prelude::rust_2021::test`.

Reply:
Would this be too paranoid for us ?

Given that Rust has three editions  since `v1` and new coming in future, together with  combinations from "std", "core" and leading colon, this is a pretty large list. 

Given the fact that `test` is there since day one, I would recommend not doing this. I checked test macros listed below and none use variants other than the two. I don't think test macros should generate other variants, if they do, let them fix it to avoid duplicated test runs but not us.

* [test-log](https://docs.rs/test-log-macros/0.2.16/src/test_log_macros/lib.rs.html#46), [test-case](https://github.com/frondeus/test-case/blob/39526d4788eea3a9acee26785fe55794875627ac/crates/test-case-core/src/test_case.rs#L102), [async-std::test](https://docs.rs/async-attributes/1.1.2/src/async_attributes/lib.rs.html#97), [googletest::test](https://docs.rs/googletest_macro/0.11.0/src/googletest_macro/lib.rs.html#141) uses `::core::prelude::v1::test`.
* [quickcheck](https://docs.rs/quickcheck/1.0.3/src/quickcheck/lib.rs.html#43-67), [test-strategy](https://github.com/frozenlib/test-strategy/blob/master/src/proptest_fn.rs#L63), [rstest](https://github.com/la10736/rstest/blob/8eec79d08dbc85a5549001c318c62772e6c42cbe/rstest_macros/src/render/mod.rs#L179) uses `#[test]`
* `proptest`,[tracing_test::traced_test](https://docs.rs/tracing-test/latest/tracing_test/attr.traced_test.html) does not generate any

Reply:
Perhaps we can match `::[std|core]::prelude::*:test`?

Reply:
Updated. Additionally, leading colons are optional now as review comments below.

Reply:
LGTM. If you can also add some of the variants to the tests, then I think this is ready to merge.